src/modules/APImethods.js

```diff
@@ -0,0 +1,30 @@
+const tvmazeApiBaseUrl = 'https://api.tvmaze.com/lookup/shows?imdb=';
+const involvementBaseUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/dnotAWJTR3nbocRUoECT/';
+
+/* eslint-disable no-await-in-loop */
+export const getShowsData = async (showsIds) => {
```

Comment:
**[Mandatory Requirement]**
- [ ] Please don't ignore the linter errors by using these inline comments because these linter errors break your application while going to the production :fearful:

Reply:
Thanks for your valuable comments on the code. We have tried a lot with /* eslint-disable no-await-in-loop */. Can you help me with this if you have some time to guide me? I will be very thankful to you.

Reply:
OK Sure let's have a zoom call dear because this error is not easily fixable :blush:

Reply:
Thanks for giving your time on our code to fix it. Thank you very much for the call and for correcting it with a few simple lines of code that made a big change to our webpage. 1000 thanks to you.